official/resnet/resnet_run_loop.py

```diff
@@ -109,6 +109,7 @@ def process_record_dataset(dataset,
   # allows DistributionStrategies to adjust how many batches to fetch based
   # on how many devices are present.
   dataset = dataset.prefetch(buffer_size=tf.data.experimental.AUTOTUNE)
+  dataset = dataset.take(1).cache().repeat()
```

Comment:
Should this behavior not be conditional on `cached_real_data` being set? I don't see that value of the flag being used anywhere. Am i missing something?

Reply:
Done.